glide.yaml

```diff
@@ -89,6 +89,9 @@ import:
 - package: github.com/spf13/pflag
   version: 4f9190456aed1c2113ca51ea9b89219747458dc1
 
+- package: go.uber.org/atomic
```

Comment:
Do we need to pull this in anymore?

Reply:
doh I remember thinking ill forget to revert that :|